goodtablesio/helpers/create.py

```diff
@@ -41,7 +41,7 @@ def create_job(validation_conf, job_id=None):
     insert_job_row(job_id)
 
     # Create celery task
-    tasks.validate.apply_async((validation_conf, job_id))
+    tasks.validate.s(validation_conf, job_id=job_id).apply_async()
```

Comment:
Why do we need to call it like this? It seems like celery always has an uglier way of doing things :)